AsyncDisplayKit/Private/ASControlTargetAction.h

```diff
@@ -0,0 +1,29 @@
+//
+//  ASControlTargetAction.h
+//  AsyncDisplayKit
+//
+//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+//
+
+#import <Foundation/Foundation.h>
+
+/**
+ @abstract ASControlTargetAction stores target action pairs registered for specific ASControlNodeEvent values.
+ ASControlNode This class stores target action pairs similar to the UIControlTargetAction class.
```

Comment:
Just to be sure, let's remove the private Apple class name. They don't like those.